packages/commonwealth/client/scripts/views/components/component_kit/cw_sidebar_menu.tsx

```diff
@@ -48,6 +48,10 @@ export const CWSidebarMenuItem = (props: MenuItem) => {
   } else if (props.type === 'community') {
     const item = props.community;
     const roles = app.roles.getAllRolesInCommunity({ chain: item.id });
+    const [isStarred, setIsStarred] = useState(
```

Comment:
this is wrong. use State cannot be called conditionally and eslint warns about that actually.

Reply:
Addressed.